### DISCUSSION TITLE
eval出现错误

### DISCUSSION THREAD

**Discussion by zhangxianbin1999** [PengfeiM/Fatigue-Driven-Detection-Based-on-CNN#38] [General]
旧代码运行eval时出现"RuntimeError: Legacy autograd function with non-static forward method is deprecated. Please use new-style autograd function with static forward method. (Example: https://pytorch.org/docs/stable/autograd.html#torch.autograd.Function)"的问题

**Comment by zyk1109** [comment]
可模仿test文件进行修改，将detection内容进行修改，具体是在第92，100行对detect内容进行修改
修改内容如下：

92：detect = Detect.apply

100：detections=detect(
		loc.view(loc.size(0),-1,4),
		softmax(conf.view(conf.size(0),-1,config.class_num)),
		torch.cat([o.view(-1,4) for o in priors],0),
		config.class_num,
		200,
		0.7,
		0.45
	).data
